File: Main-WorkCard/Kds/Common/KDSCommon/DataModels/WorkCard/WorkCardResultContainer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace KDSCommon.DataModels.WorkCard
{
    /// <summary>
    /// class for engular version
    /// </summary>

    public class WorkCardResultContainer
    {
        public SidurimWC Sidurim { get; set; }
        public HityazvutInfo FirstHityazvut { get; set; }
        public HityazvutInfo SecondHityazvut { get; set; }
        public NetuneyYomAvoda DayDetails { get; set; }
        public StatusCard CardStatus { get; set; }
        public bool  bRashemet { get; set; }
        public bool bMenahelBankShaot { get; set; }
        public clParametersDM oParams { get; set; }
    }
}

File: Main-WorkCard/Kds/KdsWebApplication/app/modules/workcard/contracts/WorkCardResultContainer.ts
module modules.workcard {

    export class WorkCardResultContainer {
        Sidurim: SidurimWC;
        FirstHityazvut: Hityatzvut;
        SecondHityazvut: Hityatzvut;
        DayDetails: {
            ZmanNesiot: {
                IsEnabled: boolean;
                Value: number;
                Attributes: AttributeField[];
            };
            Tachograf: {
                IsEnabled: boolean;
                Value: number;
                Attributes: AttributeField[];
            };
            Halbasha: {
                IsEnabled: boolean;
                Value: number;
                Attributes: AttributeField[];
            };
            Lina: {
                IsEnabled: boolean;
                Value: number;
                Attributes: AttributeField[];
            };
            HashlamaForDay: {
                IsEnabled: boolean;
                Value: number;
                Attributes: AttributeField[];
            };
            SibatHashlamaLeyom: {
                IsEnabled: boolean;
                Value: number;
                Attributes: AttributeField[];
            };
        };
        CardStatus: {
            Kodstatus: number;
            TeurStatus: string;
            ClassStr: string;
        };
        BRashemet: boolean;
        BMenahelBankShaot: boolean;
        OParams: clParametersDM;
    }

}